app/src/main/kotlin/net/sourceforge/opencamera/uploadservice/FileUploadService.kt

```diff
@@ -4,23 +4,35 @@ import android.app.NotificationChannel
 import android.app.NotificationManager
 import android.content.Context
 import android.os.Build
+import androidx.annotation.RequiresApi
+import net.gotev.uploadservice.UploadServiceConfig.initialize
 import net.gotev.uploadservice.protocols.multipart.MultipartUploadRequest
+import net.sourceforge.opencamera.BuildConfig
 import net.sourceforge.opencamera.MainActivity
 import java.io.File
 
 class FileUploadService(private val mMainActivity: MainActivity) {
```

Comment:
Try to exclude main activity or provide deps directly if you need smth (if its too hard we will think about keeping activity but it is not a good style)